src/pq_flash_index.cpp

```diff
@@ -932,14 +936,21 @@ namespace diskann {
       }
     }
 
-    compute_dists(&best_medoid, 1, dist_scratch);
+    std::vector<_u32> init_nodes;
+    init_nodes.reserve(1);
```

Comment:
can we move this this malloc to scratch.h/cpp? OR even better, use an existing scratch variable?